test/test_fsdp2/test_fsdp2_common.py

```diff
@@ -1,12 +1,12 @@
 import contextlib
-from typing import List, Type
+from typing import List
 
 import float8_experimental.config as config
 
 import torch
 import torch.distributed as dist
 import torch.nn as nn
-from float8_experimental.float8_linear import Float8Linear
+from float8_experimental.fsdp_utils import precompute_float8_amax_for_fsdp
```

Comment:
I might be very confused, but where is this `fsdp_utils` file?
![Screenshot 2024-07-09 at 7 20 22 PM](https://github.com/pytorch-labs/float8_experimental/assets/31054793/2d122295-b023-4e82-b6bd-1341493a4fce)

and a nit: maybe should be consistent in whether `pre_compute` vs. `precompute` across the PR
(either way is fine to me, `precompute` maybe since it is more concise)

Reply:
> I might be very confused, but where is this fsdp_utils file?

just added `float8_experimental/fsdp_utils.py`

> aybe should be consistent in whether pre_compute vs. precompute across the PR

good suggestion. I will use `precompute` consistently